pages/dkp/kommander/1.3/addons/requirements/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: layout.pug
+navigationTitle: Configuration Requirements
+title: Configuration Requirements
```

Comment:
```suggestion
title: Platform Service Requirements
```

Maybe rename this to just `Platform Service Requirements` as the page exclusively talks about KBA and ClusterAddons?

I think `Configuration Requirements` in this context is misnamed. 

`Configuration Requirements` is a great title for the layout of the repositories, and how the `metadata.labels` and `metadata.annotations` are needed to create a addon repository.